app/controllers/decidim_controller.rb

```diff
@@ -1,3 +1,10 @@
 # frozen_string_literal: true
 class DecidimController < ActionController::Base
+  before_action :set_raven_context
+
+  def set_raven_context
+    return unless Rails.env.production?
```

Comment:
Sure: I don't want the code to depend on what Rails environment the app is deployed. As an example, a Pull Request app environment could also be `production` but I don't want to have Sentry enabled there.

Instead of relying on Rails' environment we should rely on configuration:

`return unless Rails.application.secrets.sentry_enabled?`

And then configure it at secrets.yml per environment or with ENV vars.